tests/quantization/test_saturation_issue_export.py

```diff
@@ -138,14 +141,81 @@ def test_without_saturation_helper(target_device):
         test_without_saturation_helper(device)
 
 
-def test_are_onnx_exported_weights_tensors_clipped(tmp_path):
+class EightConvTestModel(nn.Module):
+    def __init__(self, in_out_ch):
+        super().__init__()
+        self.features = []
+        self.features.append(nn.Sequential(create_conv(*in_out_ch[0], 2, -1, -2)))
+        self.features.append(nn.Sequential(create_conv(*in_out_ch[1], 5, 1, 1)))
+        self.features.append(nn.Sequential(create_conv(*in_out_ch[2], 1, 2, 2)))
+        self.features.append(nn.Sequential(create_conv(*in_out_ch[3], 9, -1, 0)))
+        self.features.append(nn.Sequential(create_conv(*reversed(in_out_ch[3]), 3, 0, 1)))
+        self.features.append(nn.Sequential(create_conv(*reversed(in_out_ch[2]), 1, -1, 9)))
+        self.features.append(nn.Sequential(create_conv(*reversed(in_out_ch[1]), 2, 10, 1)))
+        self.features.append(nn.Sequential(create_conv(*reversed(in_out_ch[0]), 1, 1, 1)))
+        self.features = nn.Sequential(*self.features)
+
+    def forward(self, x):
+        return self.features(x)
+
+
+def test_are_fq_exported_per_channel_weights_tensors_clipped(tmp_path):
+    in_out_ch = [[1, 3], [3, 5], [5, 7], [7, 10]]
+    model = EightConvTestModel(in_out_ch)
+    nncf_config = get_config_for_export_mode(False)
+    nncf_config.update({"input_info": {
+        "sample_size": [1, 1, 20, 20]
+    }})
+
+    _, compression_ctrl = create_compressed_model_and_algo_for_test(model, nncf_config)
+
+    # Update scale tensors in the model
+    quantizers = compression_ctrl.weight_quantizers.values()
+    for quantizer in quantizers:
+        quantizer.quantizer_module_ref.scale = torch.nn.Parameter(
+            torch.rand(quantizer.quantized_module.weight.shape[0], dtype=torch.float32, requires_grad=True))
+
+    onnx_checkpoint_path = str(tmp_path / 'model.onnx')
+    compression_ctrl.export_model(onnx_checkpoint_path, input_names=['input'])
+
+    onnx_model = onnx.load(onnx_checkpoint_path)
+
+    # Find weight tensors in ONNX model
+    onnx_conv_weight_tensors = []
+
+    # Add conv weights tensors
+    # pylint:disable=no-member
+    for weight_tensor in onnx_model.graph.initializer:
+        if 'weight' in weight_tensor.name:
+            onnx_conv_weight_tensors.append(numpy_helper.to_array(weight_tensor))
+
+    # Add last convolution weights
+    for node in onnx_model.graph.node:
+        try:
+            if node.op_type == 'Constant':
+                if node.attribute[0].t.dims[0] == 1 and node.attribute[0].t.dims[1] == 3 and \
+                        node.attribute[0].t.dims[2] == 1 and node.attribute[0].t.dims[3] == 1:
+                    onnx_conv_weight_tensors.append(numpy_helper.to_array(node.attribute[0].t))
```

Comment:
I think that this code (and the `for` loop above it) should properly check for '`Conv`' node types and only consider these, instead of relying on the tensor name and dimensions.

Reply:
Unfortunately, `Conv `node doesn't have weights in attributes; it only has input names which are also useless in finding weight tensors.

Reply:
You should use the approach that is similar to what is being done in `resolve_constant_node_inputs_to_values` in test_unified_scales.py.

Reply:
Done

Reply:
Done